File: exam-slammr-webapp/src/main/java/uk/co/claritysoftware/exam/slammr/webapp/web/exception/ExamSlammrUserProfileNotFoundException.java
package uk.co.claritysoftware.exam.slammr.webapp.web.exception;

import uk.co.claritysoftware.exam.slammr.webapp.service.model.user.ExamSlammrUserProfile;

/**
 * RuntimeException for when an {@link ExamSlammrUserProfile} can not be found
 */
public class ExamSlammrUserProfileNotFoundException extends RuntimeException {

	public ExamSlammrUserProfileNotFoundException(String userId) {
		super("ExamSlammrUserProfile with id " + userId + " not found");
	}
}

File: exam-slammr-webapp/src/main/kotlin/uk/co/claritysoftware/exam/slammr/webapp/web/exception/ExamSlammrUserProfileNotFoundException.kt
package uk.co.claritysoftware.exam.slammr.webapp.web.exception

import uk.co.claritysoftware.exam.slammr.webapp.service.model.user.ExamSlammrUserProfile

/**
 * RuntimeException for when an [ExamSlammrUserProfile] can not be found
 */
class ExamSlammrUserProfileNotFoundException(userId: String) : RuntimeException("ExamSlammrUserProfile with id $userId not found")
